spec/models/ingredient_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Ingredient, type: :model do
+
+  describe "validations" do
+      it {should validate_presence_of :name}
+      it {should validate_presence_of :calories}
+  end
+
+  describe "relationships" do
+    it {should have_many :dish_ingredients}
+    it {should have_many(:dishes).through(:dish_ingredients)}
+  end
```

Comment:
Nice attention to the relationship and instance method tests in the models! Great work